include/dgl/kernel.h

```diff
@@ -42,10 +46,16 @@ void SpMM(
  * @param ufeat The source node feature.
  * @param vfeat The destination node feature.
  * @param out The output feature on edge.
+ * @param lhs_target Type of `ufeat`. (0: source; 1: edge; 2: destination)
+ * @param rhs_target Type of `vfeat`. (0: source; 1: edge; 2: destination)
  */
 void SDDMM(
-    const std::string& op, HeteroGraphPtr graph, NDArray ufeat, NDArray efeat,
```

Comment:
Does this mean that the use of `efeat` was a mistake?

Reply:
Yes. It should be `vfeat` as in its definition code.